source/ocsp-support/ocsp-support.rst

```diff
@@ -134,28 +136,69 @@ Drivers SHOULD validate OCSP Responses in the manner specified in `RFC
 6960: 3.2 <https://tools.ietf.org/html/rfc6960#section-3.2>`__ to the
 extent that their TLS library allows.
 
+MongoClient Configuration
+--------------------------
+
+This specification introduces the client-level configuration option
+defined below. Drivers that can, on a per MongoClient basis, disable
+non-stapled OCSP while keeping stapled OCSP enabled MUST implement
+this option.
+
+tlsDisableOcspEndpointCheck
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This boolean option determines whether MongoClient should refrain from
+reaching out to an OCSP endpoint i.e.  whether non-stapled OCSP should
+be disabled.  When set to true, a driver MUST NOT reach out to OCSP
+endpoints. When set to false, a driver MUST reach out to xOCSP
+endpoints if needed (as described in :ref:`Suggested OCSP Behavior`).
+This option MUST default to false.
+
+Naming Deviations
+^^^^^^^^^^^^^^^^^^
+
+Drivers MUST use the defined name of ``tlsDisableOcspEndpointCheck``
+for the connection string parameter to ensure portability of
+connection strings across applications and drivers. If drivers solicit
+MongoClient options through another mechanism (e.g. an options
+dictionary provided to the MongoClient constructor), drivers SHOULD
+use the defined name but MAY deviate to comply with their existing
+conventions. For example, a driver may use
+``tls_disable_ocsp_endpoint_check`` instead of
+``tlsDisableOcspEndpointCheck``.
+
 How OCSP interacts with existing configuration options
 ------------------------------------------------------
 
 The following requirements apply only to drivers that are able to
 enable/disable OCSP on a per MongoClient basis.
 
-1. If a connection string specifies `tlsInsecure=true` then the driver
-   MUST disable OCSP.
+1. If a connection string specifies ``tlsInsecure=true`` then the
+   driver MUST disable OCSP.
+
+2. If a connection string contains both ``tlsInsecure`` and
+   ``tlsDisableOcspEndpointCheck`` then the driver MUST throw an
+   error.
 
-2. If a driver supports ``tlsAllowInvalidCertificates``, and a
+3. If a driver supports ``tlsAllowInvalidCertificates``, and a
    connection string specifies ``tlsAllowInvalidCertificates=true``,
    then the driver MUST disable OCSP.
 
+4. If a driver supports ``tlsAllowInvalidCertificates``, and a
+   connection string specifies both ``tlsAllowInvalidCertificates``
+   and ``tlsDisableOcspEndpointCheck``, then the driver MUST
+   throw an error.
+
 The remaining requirements in this section apply only to drivers that
 expose an option to enable/disable certificate revocation checking on a
 per MongoClient basis.
 
-1. Driver MUST enable OCSP support (with stapling if possible) when this
-   option is enabled.
+1. Driver MUST enable OCSP support (with stapling if possible) when
+   this option is enabled.
 
 2. Drivers SHOULD throw an error if ``tlsInsecure=true`` or
-   ``tlsAllowInvalidCertificates=true`` are specified alongside the
+   ``tlsAllowInvalidCertificates=true`` or
```

Comment:
Minor nit: it reads a bit easier to me as "Drivers SHOULD throw an error if any of `tlsInsecure=true`, `tlsAllowInvalidCertificates=true`, or `tlsDisableOcspEndpointCheck=true` is specifed...", but this is pretty subjective, so feel free it ignore if you disagree